### ISSUE TITLE
Neuromyotonia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7641]
**Preferred term label:**
Neuromyotonia

**Synonyms**
Peripheral nerve hyperexcitability

**Definition (free text, please give PubMed ID)**
Peripheral nerve hyperexcitability manifesting as spontaneous discharges originating from motor axons or their terminals, lead to overactivity of muscles, typically manifesting as twitches, cramps and stiffness.
PMID:29554580

**Parent term (use hpo.jax.org/app)**
Abnormal peripheral nervous system physiology HP:0032120

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 137200 
PMID: 22961002  44/46

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
